backend/src/routes/user.route.ts

```diff
@@ -1,4 +1,5 @@
 import express from 'express';
+import { uploadImage } from '../services';
```

Comment:
Please import it as a method of service, not directly from services

```suggestion
import * as cloudinaryController from '../services/cloudinary';
```
or
```suggestion
import * as cloudinaryService from '../services/cloudinary';
```